modules/customLoggingApi/apiGateway.tf

```diff
@@ -36,4 +37,33 @@ resource "aws_api_gateway_integration_response" "http200" {
   depends_on = [
     aws_api_gateway_integration.sqs-integration
   ]
+}
+
+resource "aws_api_gateway_deployment" "custom_log_api_deployment" {
+  rest_api_id = aws_api_gateway_rest_api.logging_gateway.id
+  stage_name = "production"
```

Comment:
Shouldn't this change per environment?

Reply:
Also I think we should raise a wider discussion around stages in API gateway as this can become an issue.

Reply:
Yeah I see what you were referring to with the double handling of environments.
Think it's ok that we just have production for each of our environments.